Userland/Libraries/LibWeb/HTML/HTMLInputElement.cpp

```diff
@@ -1209,7 +1209,7 @@ String HTMLInputElement::covert_number_to_string(double input) const
 WebIDL::ExceptionOr<double> HTMLInputElement::value_as_number() const
```

Comment:
Re: the commit message, I think you mean "split off" (not of)? Let's reword it to be more clear, something like:
LibWeb: Move logic to check if valueAsNumber applies to its own function